SETUP.md

```diff
@@ -119,12 +119,10 @@ To install all three environments:
 
 </details>
 
-### Register the conda environment as a kernel in Jupyter
+### Using the conda environment as a kernel in Jupyter
```

Comment:
Since you did a PR about this, there have been some users that were not aware that the DSVM has jupyterhub which is accessible from port 8000. Some people are starting jupyter kernels, which is confusing. I think we should mention jupyterhub, how to register the new kernel and how to connect to the notebooks